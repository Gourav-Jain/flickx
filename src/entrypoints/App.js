import React from 'react';
import { Container, Divider } from 'semantic-ui-react';
import { HeaderBar, Dashboard } from '../components';


function App() {
  return (
    <div className="App">
      <HeaderBar />
      <Container>
        <Dashboard />
      </Container>
      <Divider />

    </div>
  );
}

export default App;
