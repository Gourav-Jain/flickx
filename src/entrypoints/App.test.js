import React from "react";
import { shallow } from "enzyme";

import App from "./App";


describe("<App />", () => {
  const shallowComponent = () => shallow(<App />);

  describe("@renders", () => {
    test("as default", () => {
      expect(shallowComponent()).toMatchSnapshot();
    });
  });
});
