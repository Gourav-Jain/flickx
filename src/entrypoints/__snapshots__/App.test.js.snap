// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> @renders as default 1`] = `ShallowWrapper {}`;
